mlflow/tracking/_tracking_service/client.py

```diff
@@ -94,18 +94,18 @@ def create_run(self, experiment_id, start_time=None, tags=None):
         :return: :py:class:`mlflow.entities.Run` that was created.
         """
 
-        tags = tags if tags else {}
+        run_tags = tags if tags else {}
```

Comment:
Do we need this change?

Reply:
At first I thought so, since it could rewrite the parameter, but it does seem unnecessary. Will undo the commit!